networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -24,6 +24,15 @@ For an additional network, IP addresses are provisioned through an IP Address Ma
 
 * Applying a YAML manifest: You can manage the additional network directly by creating an `NetworkAttachmentDefinition` object. This approach allows for the chaining of CNI plugins.
 
+[NOTE]
+====
+When deploying {product-title} nodes with the multiple network interfaces on {rh-openstack-first} with OVN SDN, subnet id of the secondary interface might take precedence over the subnet id of the primary interface. In this case, remove the DNS nameserver of the secondary interface's subnet id:
```

Comment:
```suggestion
When deploying {product-title} nodes with multiple network interfaces on {rh-openstack-first} with OVN SDN, DNS settings coming from the secondary interface might take precedence over the ones of the primary interface. In this case, remove the DNS nameservers for the subnet the secondary interface is attached to:
```

Reply:
Addressed- changed the wording little bit